app/public/js/Powerup.js

```diff
@@ -54,13 +54,13 @@ class LifePowerup extends Powerup {
 LifePowerup.animation = {frames: rectangleToFrames(new Rectangle(0, 0, 32, 32), powerupHeartImage, 1)};
 
 class AirDashPowerup extends Powerup {
-    constructor(hitBox, cooldownInSeconds, durationInSeconds) {
+    constructor(hitBox, cooldownInSeconds) {
         super(hitBox, cooldownInSeconds);
-        this.durationInSeconds = durationInSeconds;
+        this.type = POWERUP_TYPE_AIRDASH;
```

Comment:
You'll have to redeclared `POWERUP_TYPE_AIRDASH`, I deleted it since it was no longer used anywhere.

Reply:
Scratch that, I was wrong, it still exists.